tests/snuba/api/endpoints/test_organization_sessions.py

```diff
@@ -1350,3 +1352,31 @@ def test_unrestricted_date_range(self):
             }
         )
         assert response.status_code == 200
+
+    @freeze_time(MOCK_DATETIME)
+    def test_release_is_empty(self):
+        self.store_session(
+            make_session(
+                self.project1, started=SESSION_STARTED + 12 * 60, release="", environment=""
+            )
+        )
+        # Empty strings are invalid values for releases and environments, but we should still handle those cases correctly at the query layer
+        response = self.do_request(
+            {
+                "project": self.project.id,  # project without users
+                "statsPeriod": "1d",
+                "interval": "1d",
+                "field": ["sum(session)"],
+                "query": 'release:"" environment:""',
```

Comment:
`release` is required and `environment` is optional, right? So should we really treat them the same? Or should we resolve `environment:""` to "has no environment tag"?

Reply:
I don't think we should treat them differently. "has no tag" is typically expressed in discover as `!tag:*`